bridge/hardhat/script/verify.sh

```diff
@@ -0,0 +1,58 @@
+#bin/bash
+
+network_flag='bsctestnet'
+
+### npx hardhat --network bsctestnet verify --constructor-args ./hardhat/etherscan/arguments.js
+# --contract contracts/zeppelin/upgradable/proxy/TransparentUpgradeableProxy.sol:TransparentUpgradeableProxy
+# 0x8c1901c031Cdf42a846c0C422A3B5A2c943F4944
+
+# is goind to install jq
+if ! command -v jq &> /dev/null
+then
+    echo "jq could not be found, it will install"
+    apt-get install jq -Y
```

Comment:
I'd maybe [use this](https://stackoverflow.com/a/8597411/3922574) to fail if we're not under Ubuntu/Debian (won't work for OSX) - I wouldn't lose time supporting other platforms, just adding an error message so the user knows what's going on

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4